client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/ExposuresAndFlights.tsx

```diff
@@ -45,9 +43,15 @@ const ExposuresAndFlights: React.FC<Props> = ({ id }: Props): JSX.Element => {
     saveExposure(data , ids);
   }
 
+  const preventEnter = (e : React.KeyboardEvent) => {
+    if(e.key === 'Enter'){
```

Comment:
`e.key === 'Enter' && e.preventDefault()` is more pretty

Reply:
fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1025/commits/960f81b6b0e40047540ec331e23f046c993a58cb)